### PR TITLE
I'd set MaxWait to a small value for local testing, but the default i…

### PR DESCRIPTION
**Pull request by ncabatoff**
…s probably better for CI.

### CODE REVIEW

**Comment by ncabatoff** [comment]
Example failure this should address: https://circleci.hashicorp.engineering/gh/hashicorp/vault-enterprise/40284#tests/containers/1
```
=== RUN   TestSSHBackend_DynamicKeyCreate
--- FAIL: TestSSHBackend_DynamicKeyCreate (24.52s)
    backend_test.go:166: Could not connect to SSH docker container: unable to dial sshd to host "127.0.0.1:32844": ssh: handshake failed: read tcp 127.0.0.1:56226->127.0.0.1:32844: read: connection reset by peer
```

File: builtin/logical/ssh/backend_test.go [hunk 1]
```diff
@@ -149,7 +149,7 @@ func prepareTestContainer(t *testing.T, tag, caPublicKeyPEM string) (func(), str
 	sshAddress := fmt.Sprintf("127.0.0.1:%s", resource.GetPort("2222/tcp"))
 
 	// exponential backoff-retry
-	pool.MaxWait = 10 * time.Second
+	//pool.MaxWait = 10 * time.Second
```
Comment (alexanderbez) [inline builtin/logical/ssh/backend_test.go:152]: why leave this commented out and not just remove it entirely?

Comment (ncabatoff) [reply builtin/logical/ssh/backend_test.go:152]: Eh, when doing local development the default maxwait is annoyingly high, and I figured I'd save someone from having to lookup how to fix it.  Probably wrong-headed, I'll remove it.

Comment (alexanderbez) [reply builtin/logical/ssh/backend_test.go]: Gotcha, maybe just leave a comment instead?

**Pull request merged**